common/helpers/__tests__/picasso.spec.js

```diff
@@ -0,0 +1,261 @@
+import { splitDaysIntoPages, calculateBedtime } from '../picasso.js'
+import dayjs from '@/common/helpers/dayjs.js'
+
+describe('splitPicassoIntoPages', () => {
+  it.each([
+    [[[], 1], []],
+    [[[], 8], []],
+    [[[], 0], []],
+    [
+      [[1, 2, 3], 2],
+      [[1, 2], [3]],
+    ],
+    [
+      [[1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15], 8],
+      [
+        [1, 2, 3, 4, 5, 6, 7, 8],
+        [9, 10, 11, 12, 13, 14, 15],
```

Comment:
Then with the comment above, you could use:
```suggestion
        createNumDays(8),
        createNumDays(9, 15),
```

Reply:
[0d398ee](https://github.com/ecamp/ecamp3/pull/3507/commits/0d398ee0de07a60e3ef04f8e28e2570428ea613e)